docker/install/ubuntu_download_csinn2_compute_lib.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+
+install_path="/opt/csi-nn2"
+
+# Clone CSI-NN2 Compute Library source code
+git clone https://github.com/T-head-Semi/csi-nn2.git ${install_path}
```

Comment:
we periodically rebuild images e.g. if we are going to update our Python version. do you guys mind pinning this version so we don't accidentally pull in new changes and break your tests in the process of updating something else?

also, possible to do a shallow clone e.g. `--depth=1` to keep the size of the CI images down?

Reply:
Thanks for advice, we are glad to do so.